src/roswire/common/nodelet_xml.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+import typing as t
+import xml.dom.minidom as dom
+
+import attr
+from loguru import logger
+
+
+@attr.s(frozen=True, auto_attribs=True)
+class NodeletLibrary:
```

Comment:
if it's not too much trouble, could you use a docstring to briefly describe these attributes?

Reply:
Done.